packages/plugins/users-permissions/server/services/users-permissions.js

```diff
@@ -3,6 +3,11 @@
 const _ = require('lodash');
 const { filter, map, pipe, prop } = require('lodash/fp');
```

Comment:
Can we replace lodash with lodash/fp here?

Reply:
![image](https://user-images.githubusercontent.com/25851739/211513141-efbd67e9-fb61-4f90-b205-e304e83dd37f.png)
Lodash FP functions are not always interchangeable with regular lodash. In this scenario, we would need to have somehow access to the values' key.

Reply:
![image](https://user-images.githubusercontent.com/25851739/211513964-84828e08-9822-4ff3-a2c0-d936b0dd0cc6.png)
One option could be to update those mismatches everywhere with custom flows like this one, but I fear that in the end, we could introduce small bugs or changes that are almost imperceptible during the refactor.